README.md

```diff
@@ -50,10 +50,7 @@ To change the `rg` case matching options use the setting `fzf-quick-open.ripgrep
     ```
 
     ```bash
-    tee -a ~/.bashrc
-    export FZF_DEFAULT_COMMAND='fd'
-    
-    (press ctrl+d)
+    export FZF_DEFAULT_COMMAND='fd' >> ~/.bashrc
```

Comment:
Does this line do anything to bashrc or should it have an echo at the beginning: `echo "export FZF_DEFAULT_COMMAND='fd'" >> ~/.bashrc`?

Reply:
good catch. you are 100% correct. I was thinking too fast